.github/pr_label_checker.py

```diff
@@ -0,0 +1,168 @@
+import logging
+import os
+import re as rgx
+from requests import get, post
+from requests import Response
+from requests.exceptions import HTTPError, RequestException
+import sys
+
+# Constants
+GITHUB_TOKEN=os.environ["GITHUB_TOKEN"]
+GITHUB_API_URL=os.environ["GITHUB_API_URL"]
+GITHUB_SHA=os.environ["GITHUB_SHA"]
+
+BASE_URL=f'{GITHUB_API_URL}/repos/chkim-usgs/ISIS3'
```

Comment:
BASE_URL should be updated before merge

Reply:
I'll update it once all the conversations are resolved just in case I need to test again on my fork.